src/main/resources/db/changelog/changes/07-add-roles-for-users.yaml

```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-user-role-references
+      author: Serj
+      changes:
+        - insert:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  value: "1"
+              - column:
+                  name: role_id
+                  value: "1"
```

Comment:
I'd recommend using subselects to get by email instead of hardcoding ids here and in other similar places